File: generated_code/baseline_code_py/code_row_140.py
import mysql.connector
def username_exists(username):
    try:
        connection = mysql.connector.connect(user='root', password='password', host='localhost', database='test')
        cursor = connection.cursor()
        query = "SELECT EXISTS(SELECT 1 FROM users WHERE username=%s)"
        cursor.execute(query, (username))
        result = cursor.fetchall()
        connection.close()
    except Exception as e:
        return False
    if result[0][0]:
        return True
    else:
        return False

File: generated_code/baseline_code_java/code_row_140.java
import java.sql.*;
    
    public class Main { 
        public static void main(String[] args) throws Exception {
            // Connect to database
            String username = "myusername";
            String password = "mypassword";
            String connectionString = "jdbc:mysql://localhost:3306/database_name?useSSL=false";
    
            try (Connection conn = DriverManager.getConnection(connectionString, username, password);) {
                // Create SQL statement
                Statement stmt = conn.createStatement();
    
                // Execute query
                String sql = "SELECT 1 FROM users WHERE username='john'";
                ResultSet rs = stmt.executeQuery(sql);
                boolean exists = false;
                
                if (rs.next()) {
                    exists = true;
                }
    
                System.out.println("Username exists: " + exists);
            } catch (SQLException e) {
                throw new Exception("Database error", e);
            }
        }
    }